R/utilities-figures.R

```diff
@@ -651,6 +651,10 @@ createPlotsFromExcel <- function(
       type = "numeric",
       valueRange = c(0.1, 10)
     ),
+    parameterFactor = list(
+      type = "numeric",
+      valueRange = c(0.01, 0.99)
+    ),
```

Comment:
Am I understanding that correctly? This will ensure that the parameterFactor is within 0.01 and 0.99. 

I would tend to think that it should not matter if the user give for example 0.1 or 10, it should just lead to the same results and plots.

Also do we really need to force a range for the input parameterFactor? I don't think there is a limit for the sensitivityCalculation

Reply:
Extended to valueRange = c(1e-16, 1e16)